interface/therapy_groups/therapy_groups_views/addGroup.php

```diff
@@ -24,14 +24,21 @@
  * @link    http://www.open-emr.org
  */
 ?>
+
+<?php $edit = acl_check("groups","gadd",false, 'write');?>
+<?php $view = acl_check("groups","gadd",false, 'view');?>
```

Comment:
to point out, using $edit and $view are just fine here since this script is only used for groups